src/dapps/5_advantages.md

```diff
@@ -2,9 +2,9 @@
 
 Here are a few xDapp features that are making an impact across blockchain technologies:
 
-- **Expanded User Base** - Rather than being limited to the users of one blockchain, any user on any blockchain in the ecosystem can interact with an xDapp.
+- **Expanded User Base** - Rather than being limited to the users of one blockchain, any user on any blockchain in the ecosystem can interact with an xDapp which Wormhole supports.
```

Comment:
no need to have a "wormhole supported" dapp? if anything id be the other way around, an xdapp that supports wormhole